capstone/src/main/webapp/oAuth.js

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Client ID from the Developer Console 
+const CLIENT_ID = 'CLIENT_ID';
+
+let authorizeButton = document.getElementById('authorize_button');
+let signoutButton = document.getElementById('signout_button');
+let submitButton = document.getElementById('submit_button');
```

Comment:
nit: Should these be changed to, for example, `AUTHORIZE_BUTTON`?

Reply:
I'll take a look - I think they should be constants, but there is a chance that the OAuth script changes them in some way.

Reply:
I had to change the order the scripts loaded in the HTML page, but these are constants now.